clevertap-core/src/main/java/com/clevertap/android/sdk/userEventLogs/UserEventLog.kt

```diff
@@ -0,0 +1,9 @@
+package com.clevertap.android.sdk.userEventLogs
```

Comment:
userEventLogs is not a conventional package name. Use "usereventlog" instead, or just "eventlog". Rename also the corresponding test package.

Reply:
fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/cc256031c40b3675611da15ca4d813b6254be65c